src/lib/flow/include/sol-flow-metatype.h

```diff
@@ -95,12 +95,12 @@ struct sol_flow_metatype_context {
      * @param ctx The metatype context.
      * @param name The file path.
      * @param buf Where the file contents should be stored.
-     * @param size The size of the buffer.
```

Comment:
I meant, in case we change to append data to the buffer you should say what I stated. Otherwise you should also add a note saying thet the buffer will be initialized by the function and all previous data will be lost/leaked